web_timeline/static/lib/vis/vis.css

```diff
@@ -0,0 +1,810 @@
+.vis .overlay {
```

Comment:
I'm not sure about this one, because the deployment with npm is harder than including the library, and you wouldn't find version problems, so I prefer this way.